components/automate-dex/web/templates/login.html

```diff
@@ -18,14 +18,14 @@
       {{ range $c := .Connectors }}
       <div class="theme-form-row">
         <a href="{{ $c.URL }}" target="_self">
-          <button class="dex-btn theme-btn-provider">
-            <span class="dex-btn-icon dex-btn-icon--{{ $c.ID }}"></span>
-            {{ if eq $c.ID "local" }}
-            <span class="dex-btn-text">Sign in as a local user</span>
-            {{ else }}
-            <span class="dex-btn-text">Sign in with {{ $c.Name }}</span>
-            {{ end }}
-          </button>
+          <div class="dex-btn theme-btn-provider">
```

Comment:
its really a link, cause it goes somewhere else, when we nest a button inside of an anchor it causes double focus when navigating with a keyboard.